schemas/multimodal/v1/contracts.proto

```
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package fiftyone.multimodal.schemas.v1;
```

Buf (1.67.0)

[error] Files with package "fiftyone.multimodal.schemas.v1" must be within a directory "fiftyone/multimodal/schemas/v1" relative to root but were in directory "schemas/multimodal/v1".

(PACKAGE_DIRECTORY_MATCH)